extensions/amp-social-share/0.1/test/test-amp-social-share.js

```diff
@@ -109,6 +109,7 @@ describe('amp-social-share', () => {
       const tShare = ins.getElementsByTagName('span')[0];
       expect(tShare).to.not.be.null;
       expect(tShare.firstChild).to.not.be.null;
+      expect(tShare.className).to.contain('amp-social-share-twitter');
```

Comment:
you will nee to add tests to make sure that the class names are properly namespaced?